tests/runtime/out_kinesis.c

```diff
@@ -1,6 +1,8 @@
 /* -*- Mode: C; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- */
 #include <fluent-bit.h>
 #include "flb_tests_runtime.h"
+#include "../../plugins/out_kinesis_streams//kinesis.h"
```

Comment:
Could you remove this include?
It seems that it's not needed to added test cases for custom ports.

Reply:
Thank you for your work!
I have made the necessary changes.